sql/updates/latest-dev.sql

```diff
@@ -3,7 +3,7 @@ ALTER TABLE _timescaledb_catalog.telemetry_metadata ALTER COLUMN include_in_tele
 ALTER TABLE _timescaledb_catalog.telemetry_metadata RENAME TO metadata;
 ALTER INDEX _timescaledb_catalog.telemetry_metadata_pkey RENAME TO metadata_pkey;
 CREATE TABLE IF NOT EXISTS _timescaledb_catalog.continuous_aggs_materialization_invalidation_log(
-    materialization_id INTEGER PRIMARY KEY
```

Comment:
shouldnt this be an alter table in the update script or not be here at all

Reply:
this is a new table for 1.4. Update does not work if I don't have the CREATE TABLE here.

Reply:
yes since its a new table the create table command should not be in latest-dev.sql at all because the actual update script for a specific version gets generated and the generated file will already include tables.sql which has the required create table command